### PR TITLE
Feature/bk 97 background shapes ie fixes

### PR DESCRIPTION
**Pull request by remydenton**

### CODE REVIEW

File: src/_patterns/02-components/05-bolt-background-shapes/src/background-shapes.scss [hunk 1]
```diff
@@ -12,9 +12,17 @@
   }
 
   &__shape {
+    // To support IE, use the "bottom padding" hack to position svgs within the grid.
+    // See https://css-tricks.com/scale-svg/#article-header-id-10
+    position: relative;
+    height: 0;
+    padding-bottom: 100%;
 
     svg {
```
Comment (sghoweri) [inline src/_patterns/02-components/05-bolt-background-shapes/src/background-shapes.scss]: @remydenton I'm always on the fence for these sorts of selectors (especially for anything but the generic and element layers in ITCSS).

Ideally even things like this SVG element would have a class associated with it keep the CSS selector flat and not HTML element specific -- ie. `.c-bolt-background-shapes__svg`. Are there any technical reasons why we couldn't do that here?

Thoughts?

File: src/_patterns/02-components/05-bolt-background-shapes/src/background-shapes.scss [hunk 2]
```diff
@@ -12,9 +12,17 @@
   }
 
   &__shape {
+    // To support IE, use the "bottom padding" hack to position svgs within the grid.
+    // See https://css-tricks.com/scale-svg/#article-header-id-10
```
Comment (sghoweri) [inline src/_patterns/02-components/05-bolt-background-shapes/src/background-shapes.scss]: Oh I remember this one all too well ;) - nice find

File: src/_patterns/02-components/05-bolt-background-shapes/src/background-shapes.scss (hunk 1 above)
Comment (remydenton) [reply src/_patterns/02-components/05-bolt-background-shapes/src/background-shapes.scss]: Yep, totally agree.  The one catch at this point is that there is already a [.c-bolt-background-shapes__svg](https://github.com/bolt-design-system/bolt/blob/01c7d40b7beac537c11d0a33126efad51effa73a/src/_patterns/02-components/05-bolt-background-shapes/src/background-shapes.scss#L29), but that's just laziness-- I will refactor to get the element selector replaced with a class.

**Comment by remydenton** [comment]
This should be all fixed up @sghoweri.

**Comment by sghoweri** [comment]
BOOM - thanks @remydenton!!

**Pull request merged**